src/components/FeatureAnnouncementBanner/FeatureAnnouncementBanner.test.jsx

```diff
@@ -70,7 +70,7 @@ describe('<FeatureAnnouncementBanner />', () => {
     });
     expect(container.textContent).toContain('This is a test notification.');
 
-    const closeButton = document.getElementsByClassName('close')[0];
+    const closeButton = document.getElementsByClassName('btn btn-tertiary btn-sm')[0];
```

Comment:
Just to get it in writing, this test will still fail in the future if Paragon's `Alert` changes the underlying button class names in the future. We are intentionally deferring to the future on refactoring these tests to use RTL to account for it.

Reply:
Adam I would assume that this is something that is spread around in many places in the code? Could you link me more information about RTL?

Reply:
@macdiesel  Yeah, tests broke in other MFEs due to the `Alert` changes going to from an "X" button to a "Dismiss" button to close the alert because the tests select elements based on things that are implementation details of the Paragon component itself.

Instead, we'd ideally querying for the close button to test dismiss behavior based on the text of the button itself now rather than a class name.

https://testing-library.com/docs/react-testing-library/intro/#the-problem